ide/bugtracking.bridge/src/org/netbeans/modules/bugtracking/vcs/HookUtils.java

```diff
@@ -43,16 +43,10 @@ public class HookUtils {
      * @return
      */
     public static String prepareFormatString(String formatString, String... keys) {
-        formatString = formatString.replaceAll("\\{", "'\\{'");                 // NOI18N
-        formatString = formatString.replaceAll("\\}", "'\\}'");                 // NOI18N
-        for (int i = 0; i < keys.length; i++) {
-            String key = keys[i];
-            formatString =
-                formatString.replaceAll(
-                    "'\\{'" + key + "'\\}'",                                    // NOI18N
-                    "\\{" + i + "\\}"                                           // NOI18N
-                );
-
+        formatString = formatString.replace("{", "'{'");                 // NOI18N
+        formatString = formatString.replace("}", "'}'");                 // NOI18N
+        for (String key : keys) {
+            formatString = formatString.replace("'{'" + key + "'}'", "{" + key + "}");
         }
```

Comment:
second `key` should be `i` but you removed the index from the loop, so you have to revert that too.

Reply:
fixed